app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_save :update_posts_counter
+
+  private
+
+  def update_posts_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
[OPTIONAL] Please rename the method name with `update_like_counter` because this is for like count not for post count so it will look more understandable :slightly_smiling_face: